payments-core/src/main/java/com/stripe/android/view/BecsDebitBanks.kt

```diff
@@ -7,7 +7,7 @@ import kotlinx.parcelize.Parcelize
 import org.json.JSONObject
 import java.util.Scanner
 
-internal class BecsDebitBanks(
+class BecsDebitBanks(
```

Comment:
After other refactors let's check and make sure this is still required to be exported.  If it is we should add the RestrictTo(RestrictTo.Scope....) annotation.  By adding the annotation this function it will be clear to merchant developers that this function is not intended for public use.   The annotation will result in no public documentation, and it won't auto-complete in the merchant's IDE.